great_expectations/cli/suite.py

```diff
@@ -430,15 +452,15 @@ def _suite_scaffold(suite: str, directory: str, jupyter: bool) -> None:
     renderer = SuiteScaffoldNotebookRenderer(context, _suite, batch_kwargs)
     renderer.render_to_disk(notebook_path)
 
+    send_usage_message(data_context=context, event=usage_event, success=True)
+
```

Comment:
Need to send message before launching jupyter notebook. Or else the message is never sent